utils/test/visibility_tests/LinearLineOfSightTestCase.py

```diff
@@ -161,6 +164,38 @@ def test_linear_line_of_sight(self):
         #TODO: check attached profile graphs were created
 
         return
+
+    def test_empty_observer(self):
+        toolOutput = None
+        # Check for empty observer input parameter
+        with self.assertRaises(arcpy.ExecuteError) as manage_raise:
+            toolOutput = arcpy.LinearLineOfSight_mt(self.observers_empty,
```

Comment:
Great example of providing bad input to tool.  FYI @topowright @ACueva @dfoll - an example to keep in mind for later when you want to start doing this kind of testing.